### DISCUSSION TITLE
Error: "Failed to load private key from <inline>, Cause: error:0b000074:X.509 certificate routines:OPENSSL_internal:KEY_VALUES_MISMATCH"

### DISCUSSION THREAD

**Discussion by kjidev** [istio/istio#48716] [General]
Hello everyone, 

I've installed and setup istio version 1.19.3 in a private Azure Kubernetes Cluster (AKS). Currently, the istio-ingressgateway pod logs show error:

`"warning	envoy config external/envoy/source/extensions/config_subscription/grpc/grpc_subscription_impl.cc:128	gRPC config for type.googleapis.com/envoy.config.listener.v3.Listener rejected: Error adding/updating listener(s) 0.0.0.0_8181: Failed to load private key from <inline>, Cause: error:0b000074:X.509 certificate routines:OPENSSL_internal:KEY_VALUES_MISMATCH"`

Restarted the ingress gateway pod though it results with the same error. 

Anyone know how to resolve this issue?

**Comment by howardjohn** [comment]
This is likely the Secret your certificate is stored in is an invalid
